packages/core/test/insured-bridge/BridgePool.js

```diff
@@ -931,16 +931,8 @@ describe("BridgePool", () => {
       );
       assert(await didContractThrow(bridgePool.methods.settleRelay(depositData).send({ from: relayer })));
 
-      // Expire and settle proposal on the OptimisticOracle.
+      // Set time such that optimistic price request is settleable.
       await timer.methods.setCurrentTime(expectedExpirationTimestamp).send({ from: owner });
-      await optimisticOracle.methods
```

Comment:
Before we were manually calling `settle` to pay out the slow relayer, this PR removes that need.